controller/src/main/java/org/jboss/as/controller/capability/CapabilityServiceSupport.java

```diff
@@ -86,6 +88,21 @@ public NoSuchCapabilityException(String message) {
      */
     <T> T getCapabilityRuntimeAPI(String capabilityBaseName, String dynamicPart, Class<T> apiType) throws NoSuchCapabilityException;
```

Comment:
For consistency, this variant should have an Optional version too.

Reply:
I added the missing variant using optional, it is still pending for a review from Jeff.